tajo-client/src/main/java/org/apache/tajo/jdbc/TajoResultSetBase.java

```diff
@@ -364,10 +364,16 @@ private Timestamp getTimestamp(Tuple tuple, TimeZone tz, int index) throws SQLEx
   }
 
   private Timestamp toTimestamp(TimeMeta tm, TimeZone tz) {
+
+    long javaTime = DateTimeUtil.julianTimeToJavaTime(DateTimeUtil.toJulianTimestamp(tm));
```

Comment:
It looks inconvenient to call both functions whenever we want to convert TimeMeta into java time. How about adding a function which simply wraps these calls?